tests/Custom/TestCase.php

```diff
@@ -37,4 +37,16 @@ protected function getConfig(): WebdriverClassicConfig
     {
         return WebdriverClassicConfig::getInstance();
     }
+
+    /**
+     * @before
+     *
+     * @return void
+     */
+    protected function checkSkippedTest()
```

Comment:
Please use `: void` and remove that part from PHPDoc.

Reply:
Done.